test/customizations-smtp.yml

```diff
@@ -0,0 +1,74 @@
+forge:
```

Comment:
Not quite sure what we need this file for?

Reply:
It's some extra tests, I'll remove it